virt/virt-2-4-release-notes.adoc

```diff
@@ -12,10 +12,20 @@ include::modules/virt-what-you-can-do-with-virt.adoc[leveloffset=+2]
 == New and changed features
 //Placeholder for new content.
 
-//CNV-4686 - Container-native Virtualization x" in the SVVP catalog listing
+//CNV-4686 - Container-native Virtualization x" in the SVVP catalog listing - Jira story was obsoleted.
 
 //CNV-4687 - Ensure platform passes Windows Server Virtualization Program with RHCOS workers.
 
+* {VirtProductName} is certified in Microsoft's Windows Server Virtualization Validation Program (SVVP) to run Windows Server workloads.
+
```

Comment:
Nit: it's not breaking the build but this empty line (L20) serves no purpose and should be removed.

Reply:
Line removed.

Reply:
Removed.